subsys/bluetooth/services/Kconfig.ddfs

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig BT_DDFS
+	bool "Direction and Distance Finding Service"
+	select BT_NRF_SERVICES
```

Comment:
Consider also adding `SELECT EXPERIMENTAL` to mark the implementation as experimental.